### PR TITLE
Create lv_example_chart_9.py

### PR DESCRIPTION
**Pull request by Karijn**
lv_example_chart_9.c was exactly what i needed, thanks for that one!
the only thing is, i needed it in MicroPython :-) 
Here it is

### Description of the feature or fix
translated lv_example_chart_9.c to micro python

### Checkpoints
- [ ] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

File: examples/widgets/chart/lv_example_chart_9.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+def add_data(t):
+    chart.set_next_value(ser, lv.rand(10, 90))
+
+    p = chart.get_point_count()
+    s = chart.get_x_start_point(ser)
+    a = chart.get_y_array(ser)
+
+    a[(s + 1) % p] = lv.CHART_POINT.NONE
+    a[(s + 2) % p] = lv.CHART_POINT.NONE
+    a[(s + 2) % p] = lv.CHART_POINT.NONE
```
Comment (amirgon) [inline examples/widgets/chart/lv_example_chart_9.py]: Why do you assign `lv.CHART_POINT.NONE` three times?

Comment (Karijn) [reply examples/widgets/chart/lv_example_chart_9.py]: i think one time is enough, but i just copied the .c example. I can adjust it though

**Comment by kisvegabor** [comment]
> Why do you assign lv.CHART_POINT.NONE three times?

In the C example I did it to have a larger gap.

**Comment by Karijn** [comment]
@kisvegabor 
What is the status of this PR? Do i have to do anything still?

**Comment by kisvegabor** [comment]
Could you add back these:
```py
    a[(s + 2) % p] = lv.CHART_POINT.NONE
    a[(s + 3) % p] = lv.CHART_POINT.NONE
```

IMO it looks better and it shows the user how to make the gap larger.

**Comment by kisvegabor** [comment]
Thank you!

**Pull request merged**